src/component/tooltip/TooltipView.ts

```diff
@@ -951,10 +955,20 @@ class TooltipView extends ComponentView {
                         && lastIdxItem.seriesIndex === newIdxItem.seriesIndex
                         && lastIdxItem.dataIndex === newIdxItem.dataIndex;
                 });
+
+                // check is cbParams data value changed
+                lastCbParamsList && zrUtil.each(lastItem.seriesDataIndices, (idxItem) => {
+                    const series = self._ecModel.getSeriesByIndex(idxItem.seriesIndex);
+                    const cbParams = series.getDataParams(idxItem.dataIndexInside) as TooltipCallbackDataParams;
```

Comment:
I think it's better to use the `cbParamsList` in parameter instead of get a new one.

Reply:
fixed